CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 
 <!-- Your comment below this -->
 
+- Add support for BitBucket Cloud Repository Access Token - [@thawankeane]
```

Comment:
did you read the preamble of changelog? you need to add your handle link bottom of the file. see the misrendering yourself:
- https://github.com/thawankeane/danger-js/blob/3ce2eae0c58311b72ae787c3af0501c70fd610af/CHANGELOG.md

Reply:
Oops! I didn't note it, sorry. Fixing it right now